src/backend/api/upload/add-resume.js

```diff
@@ -0,0 +1,112 @@
+// taken from
+//https://www.labnol.org/google-drive-api-upload-220412
+import { assert } from 'console';
+import { JWT } from 'google-auth-library';
+import {google} from 'googleapis';
+import { objOf } from 'ramda';
+import stream from 'stream';
+
+// get auth using the correct service account
+const getAuth = async () => {
+  return new JWT({
+        scopes:[
+          "https://www.googleapis.com/auth/drive.file",
+          "https://www.googleapis.com/auth/drive",
+          "https://www.googleapis.com/auth/drive.appdata",
+          "https://www.googleapis.com/auth/drive.metadata",
```

Comment:
Is it necessary to specify all of these scopes ? I think only the top-level `https://www.googleapis.com/auth/drive` is required.